Directory.Build.targets

```diff
@@ -51,30 +51,4 @@
       RemoveProperties="TargetFramework"
       StopOnFirstFailure="True" />
   </Target>
-
-  <!-- BEGIN workaround for https://github.com/dotnet/sdk/issues/43339; remove after updated to VS 17.12, future 17.11 patch or when arcade ships the the same workaround -->
```

Comment:
This arcade ships with the workaround built int. This will also fix signed builds.